### PR TITLE
Increase timeout to reduce flakiness mobile applab share eyes test

### PR DESCRIPTION
**Pull request by philbogle**

### CODE REVIEW

File: dashboard/test/ui/features/footer.feature [hunk 1]
```diff
@@ -239,7 +239,7 @@ Feature: Checking the footer appearance
 
     # Additional wait to let scroll position settle and possibly have the
     # pin-to-home-screen popup go away
-    And I wait for 10 seconds
+    And I wait for 20 seconds
```
Comment (Bjvanminnen) [inline dashboard/test/ui/features/footer.feature:242]: Hmmm...this really seems unfortunate. 10s was already a pretty long wait. That said, I don't really have any better suggestions :(

**Comment by Bjvanminnen** [comment]
lgtm

File: dashboard/test/ui/features/footer.feature (hunk 1 above)
Comment (breville) [inline dashboard/test/ui/features/footer.feature:242]: Is it possible to wait to see something?

Comment (philbogle) [inline dashboard/test/ui/features/footer.feature:242]: You can definitely watch the ipad test run and see that it takes a while for things to happen in the test environment.

I don't have the expertise to know what to wait to see, if someone who developed the code can improve the precision of the wait that would be great.

**Comment by breville** [comment]
Would this benefit from a "match timeout" increase?

http://support.applitools.com/customer/en/portal/articles/2099488-match-timeout
https://github.com/code-dot-org/code-dot-org/pull/6477

**Comment by philbogle** [comment]
That seems like it might be a helpful addition.   Would it even make sense
to make the default when looking for a match?

**Comment by breville** [comment]
@aoby what do you think?

**Comment by aoby** [comment]
I already increased it (from the default 2 to 5 seconds) across the board yesterday :) See https://github.com/code-dot-org/code-dot-org/pull/6477/files

**Comment by breville** [comment]
oh nice.. missed that it was global.  thanks.

**Pull request merged**